typescript/infra/scripts/core.ts

```diff
@@ -13,54 +16,73 @@ import {
   getCoreRustDirectory,
   getCoreVerificationDirectory,
   getEnvironment,
-  getEnvironmentDirectory,
 } from './utils';
 
+function mergeVerificationInputs<ChainName extends Chains>(
+  existingInputsMap: ChainMap<ChainName, ContractVerificationInput[]>,
+  newInputsMap: ChainMap<ChainName, ContractVerificationInput[]>,
+): ChainMap<ChainName, ContractVerificationInput[]> {
+  const allChains = new Set<ChainName>();
+  Object.keys(existingInputsMap).forEach((_) => allChains.add(_ as ChainName));
+  Object.keys(newInputsMap).forEach((_) => allChains.add(_ as ChainName));
```

Comment:
in hindsight not sure we should merge these
they are not appropriately structured to be merged